src/app/work-item/work-item-list/work-item-list-entry.component.scss

```diff
@@ -0,0 +1,22 @@
+@import "../../../assets/stylesheets/custom";
+
+.type span{
```

Comment:
I was only asking why. I know they are equivalent. I didn't know if @SMahil had done it that way for a reason or not.